crates/astria-sequencer/src/accounts/types.rs

```diff
@@ -89,7 +89,7 @@ impl std::fmt::Display for Address {
     Ord,
     Debug,
 )]
-pub struct Balance(u128);
+pub struct Balance(pub(crate) u128);
```

Comment:
why is the inner value `pub(crate)`?

Reply:
so that we can declare constants like `pub(crate) const TRANSFER_FEE: Balance = Balance(12);` and `pub(crate) const SEQUENCE_ACTION_FEE_PER_BYTE: Balance = Balance(1);`

Reply:
@itamarreif No, because trait methods are not currently usable in const expressions.